db/migrate/20190225142729_add_last_collected_date_to_ems.rb

```diff
@@ -0,0 +1,5 @@
+class AddLastCollectedDateToEms < ActiveRecord::Migration[5.0]
+  def change
+    add_column :ext_management_systems, :last_collected_date, :datetime
```

Comment:
Discussion note, _date was used here for consistency with other date columns...would have like this to be called _at otherwise.